File: source/nestjs-petclinic-rest/src/repository/vet.repository.ts
import { Repository, EntityRepository } from 'typeorm';
import { CustomRepository } from './../typeorm/typeorm-ex.decorator';
import { Vet } from './../domain/entities/vet.entity';
import { IntentMerge } from './../intent/intent.decorators';

@IntentMerge()
@CustomRepository(Vet)
export class VetRepository extends Repository<Vet> {
}

File: source/spring-petclinic-rest/src/main/java/com/spring_petclinic/spring_petclinic_rest/data/VetRepository.java
package com.spring_petclinic.spring_petclinic_rest.data;

import org.springframework.data.jpa.repository.JpaRepository;
import com.spring_petclinic.spring_petclinic_rest.domain.models.Vet;
import com.spring_petclinic.spring_petclinic_rest.intent.IntentMerge;
import java.util.UUID;

/**
 * Spring Data JPA repository for the Vet entity.
 */
@IntentMerge
public interface VetRepository extends JpaRepository<Vet, Integer>{
}